app/views/courses/_course.html.erb

```diff
@@ -12,5 +12,7 @@
 <p><label for="course_info">Course Information</label><br/>
 <%= text_field 'course', 'info'  %></p>
 
+<p><label>Default Language</label><br/>
+  <%= select("course", "locale", [["English", "en_US"], ["Hindi", "hi_IN"]], {:selected => @course ? @course.locale : ["English", "en_US"]}) %></p>
```

Comment:
same comment about locale enum

Reply:
Fixed. Added new global variable in contoller.